stress/Cargo.toml

```diff
@@ -4,11 +4,16 @@ version = "0.1.0"
 edition = "2021"
 publish = false
 
-[[bin]] # Bin to run the metrics stress tests
+[[bin]] # Bin to run the metrics stress tests for Counter
 name = "metrics"
```

Comment:
Can we just rename this with _counter suffix?

Reply:
I renamed the files, and temporarily retaining "metrics" as bin name for counters, just to make it easy to run it, given it is run most frequently. once we are done with refactoring, will use full names